### PR TITLE
Check labels for container types for resize and realloc

### PR DESCRIPTION
**Pull request by masterleinad**
Only tested for `Serial`.

### CODE REVIEW

**Comment by dalg24** [comment]
Did you check that other containers do the right thing?

**Comment by masterleinad** [comment]
> Did you check that other containers do the right thing?

I believe that unit test file covers all containers that implement `realloc/resize`. `DynRankView` needed fixing, but with these changes, the test passed for the `Serial` backend.

**Comment by masterleinad** [comment]
Running on `Cuda` revealed that we shouldn't check for `HostSpace` necessarily since the host View gets "_mirorr" appended.

File: containers/unit_tests/CMakeLists.txt [hunk 1]
```diff
@@ -13,20 +13,7 @@ foreach(Tag Threads;Serial;OpenMP;HPX;Cuda;HIP;SYCL)
...
-        DynViewAPI_rank12345
-        DynViewAPI_rank67
-        ErrorReporter
-        OffsetView
-        ScatterView
-        StaticCrsGraph
         WithoutInitializing
-        UnorderedMap
-        Vector
-        ViewCtorPropEmbeddedDim
```
Comment (dalg24) [inline containers/unit_tests/CMakeLists.txt]: I suppose you didn't mean to commit this

Comment (masterleinad) [reply containers/unit_tests/CMakeLists.txt]: Fixed.

**Pull request merged**